### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by Victor-Park**
https://user-images.githubusercontent.com/99834589/161839438-5dd16534-a61d-4ea3-a046-cc3f498b4c08.mov

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,8 +16,46 @@
...
+    <div class="container line-height-large">
+      <div class="row padding-left">
+        <h3 class="font">Code Journal</h3>
+      </div>
+    </div>
   </header>
   <main>
+    <div class="background-gray height-full">
+      <div data-view="entry-form" class="container">
+        <h2 class="column-full header-spacing">New Entry</h2>
```
Comment (codingcodymiller) [inline index.html:28]: You should never have a `column` element immediately inside a `container` element. You should make an element with the `row` class that is a child of this `container` class element, and move this `column` element into that `row`

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+
+var form = document.querySelector('form');
+
+form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var userInput = {};
+  userInput.title = form.elements.title.value;
+  userInput.url = form.elements.url.value;
+  userInput.notes = form.elements.notes.value;
+  userInput.entryId = data.entries.length + 1;
```
Comment (codingcodymiller) [inline js/main.js]: You should be using the value of `data.nextEntryId`, not the length of the entries array.

**Comment by Victor-Park** [comment]
https://user-images.githubusercontent.com/99834589/161841341-8c57df89-ea1c-4bbf-ad63-a801b07b0cd5.mov

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Victor-Park** [comment]
https://user-images.githubusercontent.com/99834589/161843533-77adc762-ee91-43a2-a100-3c69ef063522.mov

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**